desktop/apps/buy_art/stylesheets/fonts.styl

```diff
@@ -0,0 +1,169 @@
+@font-face
```

Comment:
It's also worth noting that we would normally have access to these style globally through the main layout component, but since the design doesn't include the header that comes with main layout we aren't using it which is why we've imported these style.